packages/ord-connect/src/hooks/useSignMessage.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react";
+import { useOrdContext } from "../providers/OrdContext.tsx";
+import signMessage from "../lib/signMessage.ts";
+
+export function useSignMessage(): [
+  (address, message) => Promise<string>,
+  string,
+  boolean,
+] {
+  const { network, wallet, publicKey, format } = useOrdContext();
+  const [error, setError] = useState<string | null>(null);
+  const [loading, setLoading] = useState<boolean>(false);
```

Comment:
No need to specify boolean type, it should be inferred from `false`. Also for consistency with all other libraries like react-query I suggest renaming `loading` to `isLoading`.

Reply:
updated